spec/models/post_spec.rb

```diff
@@ -224,7 +150,7 @@
   describe '#publish' do
     it 'sets the post to published = true' do
       post.publish
-      expect(post.published_at).to be
+      expect(post.published_at).not_to be_nil
```

Comment:
no need to changue, but IMO positive assertions are easier to read: `to be_present`